app/models/grda_warehouse/hud/client.rb

```diff
@@ -668,17 +664,16 @@ def cohorts_for_dashboard
 
     def last_exit_destination
       last_exit = source_exits.order(ExitDate: :desc).first
-      if last_exit
+      unless last_exit
         destination_code = last_exit.Destination || 99
         if destination_code == 17
           destination_string = last_exit.OtherDestination
         else
           destination_string = HUD.destination(destination_code)
         end
         return "#{destination_string} (#{last_exit.ExitDate})"
-      else
-        return "None"
       end
+      return 'None'
```

Comment:
This looks like an accidental inversion.  Maybe you meant `return 'None' unless last exit`
followed by the rest of the code.  I think I'd prefer to just mark this one for later processing and not change the code.

Reply:
I'll back it out.

Reply:
✅